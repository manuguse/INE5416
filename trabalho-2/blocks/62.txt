1 5 8 8 11 17 19 19
1 5 5 5 11 18 18 18
2 6 5 5 12 12 12 18
2 2 9 10 13 14 12 18
3 2 9 10 14 14 20 18
3 3 9 9 14 14 14 14
4 7 7 7 15 15 16 21
4 4 7 7 16 16 16 16