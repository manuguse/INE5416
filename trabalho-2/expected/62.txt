2 5 2 1 2 1 2 1
1 2 6 3 1 3 1 6
3 1 4 1 3 1 4 5
2 4 3 2 1 4 2 4
3 1 2 1 6 3 1 2
1 2 1 4 1 2 7 5
2 1 4 5 2 1 2 1
1 3 2 3 5 4 1 3